src/PIL/ImageOps.py

```diff
@@ -576,19 +576,20 @@ def solarize(image, threshold=128):
     return _lut(image, lut)
 
 
-def exif_transpose(image):
+def exif_transpose(image, inPlace=False):
```

Comment:
snake_case, and make it a required keyword argument (re: https://github.com/python-pillow/Pillow/pull/6954#issuecomment-1432872156)?

```suggestion
def exif_transpose(image, *, in_place=False):
```

Reply:
I've pushed a commit to make it a keyword-only argument, but regarding snake_case, would it be better to match the existing `inPlace` case of [`ImageCms.applyTransform()`](https://pillow.readthedocs.io/en/stable/reference/ImageCms.html#PIL.ImageCms.applyTransform)?

Reply:
Hmm, consistency is good, but I think these two modules are sufficiently different that we can use snake_case for new things here.